qt/python/mantidqt/widgets/codeeditor/execution.py

```diff
@@ -120,12 +120,12 @@ def execute_async(self, code_str, line_offset, filename=None, blocking=False):
             self._task = task
             return task
         else:
-            self._task = BlockingAsyncTaskWithCallback(self.execute, args=(code_str, line_offset, filename),
+            self._task = BlockingAsyncTaskWithCallback(self.execute, args=(code_str, filename, line_offset),
```

Comment:
This is good. On further investigation though the `AsyncTask` call above needs changing too but I can't attach a comment to a line that hasn't changed.